packages/api/AlvTimeWebApi/Controllers/HolidayController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using AlvTime.Business;
+using AlvTimeWebApi.Controllers.Utils;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+
+namespace AlvTimeWebApi.Controllers
+{
+    [Route("api")]
+    [ApiController]
+    public class HolidayController : Controller
```

Comment:
Suuuuuuuuuper uviktig detalj, men det holder å arve fra `ControllerBase` i stedet for `Controller`. `Controller` inneholder en del støttemetoder for views, så det er faktisk anbefalt å arve fra `ControllerBase` når en holder på med et API. For praktisk betydning har dette null og niks å si.

- https://stackoverflow.com/questions/55239380/why-derive-from-controllerbase-vs-controller-for-asp-net-core-web-api
- https://docs.microsoft.com/en-us/aspnet/core/web-api/?view=aspnetcore-3.1